tests/core/test_cluster_auth_tokens.py

```diff
@@ -34,15 +34,15 @@ def test():
 
 
 # as an admin, we should have access
-@pytest.mark.skip(reason='cluster testing needs refactor')
+# @pytest.mark.skip(reason='cluster testing needs refactor')
```

Comment:
are you unskipping the test?

Reply:
Just to make sure changes are working properly I temporarily enable them for Drone, will remove this commit when the vendor is updated & WIP removed